### PR TITLE
Text Input component

### PR DESCRIPTION
**Pull request by matikrk**

### CODE REVIEW

File: src/components/form-elements/TextInput.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const TextInput = props => {
+  const {
+    type = 'text',
+    spacedBottom = true,
+    valid = VALIDATION.UNDEFINED,
+    size = SIZE.NORMAL,
+    color = COLOR.NORMAL,
+    fullWidth,
+    value,
+    ...additionalProps
```
Comment (aju) [inline src/components/form-elements/TextInput.jsx]: 👍

File: src/components/form-elements/TextInput.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  } = props;
+
+  const inputClass = classnames('sg-input', {
+    'sg-input--spaced-bottom': spacedBottom,
```
Comment (aju) [inline src/components/form-elements/TextInput.jsx]: do we still use this option ? :(

Comment (matikrk) [inline src/components/form-elements/TextInput.jsx]: 3 places:
brainly/openstudy-search-landing-page  landing-toplayer-email.hbs
brainly/srv-mirko-generator – index.html
brainly/symfony – splash_screen.html.twig

Comment (aju) [inline src/components/form-elements/TextInput.jsx]: I think we could fix brainly/symfony – splash_screen.html.twig and don't care about the rest :D @kdzwinel @vergilius WDYT?

Comment (matikrk) [inline src/components/form-elements/TextInput.jsx]: I can delete it from component and create issue in styleguide to delete it from css

Comment (kdzwinel) [inline src/components/form-elements/TextInput.jsx]: no need to create an issue in symfony. just bump the major version and leave info in tag notes

Comment (matikrk) [inline src/components/form-elements/TextInput.jsx]: 👍

**Pull request merged**